### PR TITLE
Added support for the Shisaku keyboard.

### PR DESCRIPTION
**Pull request by ADPenrose**
## Description

Added support for the Shisaku keyboard; a 40% alice-like keyboard, assembled only with THT components, featuring a solenoid that brings back that cozy typewriter feel.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* As this is the first PR regarding the kintsugi keyboard, no issue has been fixed or closed yet.

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/adpenrose/shisaku/readme.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Shisaku
+
+![shisaku](https://i.imgur.com/42H1h1d.jpg)
```
Comment (drashna) [inline keyboards/adpenrose/shisaku/readme.md:3]:
```suggestion
![shisaku](https://i.imgur.com/42H1h1dh.jpg)
```

**Review by drashna — APPROVED** [review]

File: keyboards/adpenrose/shisaku/info.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (fauxpark) [inline keyboards/adpenrose/shisaku/info.json:1]: Please redo the layout without rotation.
image

File: keyboards/adpenrose/shisaku/config.h [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
+
+/* disable these deprecated features by default */
+// #define NO_ACTION_MACRO
+// #define NO_ACTION_FUNCTION
+
```
Comment (fauxpark) [inline keyboards/adpenrose/shisaku/config.h:92]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by ADPenrose** [comment]
Were the added changes sufficient and correct?

File: keyboards/adpenrose/shisaku/shisaku.h [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+) { \
+    { K00, K01, K02, K03, K04, K05, K06 }, \
+    { K10, K11, K12, K13, K14, K15, K16 }, \
+    { K20, K21, K22, K23, K24, K25, K26 }, \
+    { K30, KC_NO, KC_NO, K33, K34, K35 }, \
+    { K40, K41, K42, K43, K44, K45, KC_NO}, \
+    { K50, K51, K52, K53, K54, K55, KC_NO }, \
+    { K60, K61, K62, K63, K64, K65, KC_NO }, \
+    { K70, KC_NO, KC_NO, KC_NO, K74, K75, KC_NO } \
+}
```
Comment (fauxpark) [inline keyboards/adpenrose/shisaku/shisaku.h:30]:
```suggestion
#define XXX KC_NO

#define LAYOUT_all( \
         k00, k01, k02, k03, k04, k05,    k06, k10, k11, k12, k13, k14, \
         k15, k16, k20, k21, k22, k23,    k24, k25, k26, k30, k33, k34, \
    k35, k40, k41, k42, k43, k44, k45,    k50, k51, k52, k53, k54, k55, k60, \
    k61,                k62, k63, k64,         k65,           k70, k74, k75 \
) { \
    { k00, k01, k02, k03, k04, k05, k06 }, \
    { k10, k11, k12, k13, k14, k15, k16 }, \
    { k20, k21, k22, k23, k24, k25, k26 }, \
    { k30, XXX, XXX, k33, k34, k35, XXX }, \
    { k40, k41, k42, k43, k44, k45, XXX }, \
    { k50, k51, k52, k53, k54, k55, XXX }, \
    { k60, k61, k62, k63, k64, k65, XXX }, \
    { k70, XXX, XXX, XXX, k74, k75, XXX } \
}
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by ADPenrose** [comment]
Compiled and flashed on different hardware and everything works just as expected.

**Comment by ADPenrose** [comment]
Were the added changes correct?

**Review by drashna — APPROVED** [review]

File: keyboards/adpenrose/shisaku/info.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+    "keyboard_name": "shisaku",
+    "url": "https://github.com/ADPenrose/shisaku_keeb",
+    "maintainer": "ADPenrose",
```
Comment (fauxpark) [inline keyboards/adpenrose/shisaku/info.json:4]:
```suggestion
    "keyboard_name": "Shisaku",
    "manufacturer": "adpenrose"
    "url": "https://github.com/ADPenrose/shisaku_keeb",
    "maintainer": "ADPenrose",
    "usb": {
        "vid": "0x4450",
        "pid": "0x0003",
        "device_version": "0.0.1"
    },
```

File: keyboards/adpenrose/shisaku/config.h [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x4450 // "ADP"
+#define PRODUCT_ID   0x0003 // 'Cause it's my third keeb
+#define DEVICE_VER   0x0001
+#define MANUFACTURER adpenrose
+#define PRODUCT      Shisaku
+
```
Comment (fauxpark) [inline keyboards/adpenrose/shisaku/config.h:14]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by ADPenrose** [comment]
Were the added changes right?

**Review by drashna — APPROVED** [review]

**Comment by ADPenrose** [comment]
Do I need to modify something else so that this keyboard is merged?

File: keyboards/adpenrose/shisaku/config.h [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+/* disable debug print */
+//#define NO_DEBUG
+
+/* disable print */
+//#define NO_PRINT
+
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
```
Comment (keyboard-magpie) [inline keyboards/adpenrose/shisaku/config.h:80]: Most of this does not need to be included in the default keymap (especially NKRO, seeing as this is a VUSB board!)

**Review by keyboard-magpie — CHANGES_REQUESTED** [review]

File: keyboards/adpenrose/shisaku/config.h (hunk 7 above)
Comment (ADPenrose) [reply keyboards/adpenrose/shisaku/config.h:80]: I'll change it ASAP

**Comment by ADPenrose** [comment]
Is everything correct?

**Review by keyboard-magpie — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**